modules/rosa-create-objects.adoc

```diff
@@ -0,0 +1,441 @@
+
+|--url
+|(LDAP) An RFC 2255 URL (string) which specifies the LDAP search parameters to use.
+
+|--username-attributes
+|(LDAP) The list (string) of attributes whose values should be used as the preferred username. Default: uid
+|===
+
+.OpenID arguments
+|===
+|Option |Definition
+
+|--email-claims
+|(OpenID) The list (string) of claims to use as the email address.
+
+|--extra-scopes
+|(OpenID) The list (string) of scopes to request, in addition to the 'openid' scope, during the authorization token request.
+
+|--issuer-url
+|(OpenID) The URL (string) that the OpenID provider asserts as the issuer identifier. It must use the HTTPS scheme with no URL query parameters or fragment.
+
+|--name-claims
+|(OpenID) The list (string) of claims to use as the display name.
+
+|--username-claims
+|(OpenID) The list (string) of claims to use as the preferred username when provisioning a user.
+|===
+
+.Optional arguments inherited from parent commands
+|===
+|Option |Definition
+
+|-h, --help
+|Shows help for this command.
+
+|debug
+|Enables debug mode.
+
+|-i, --interactive
+|Enables interactive mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|-v, --v level
+|Log level for V logs.
+|===
+
+.Examples
+Add a GitHub identity provider to a cluster named 'mycluster'.
+
+[source,terminal]
+----
+$ rosa create idp --type=github --cluster=mycluster
+----
+
+Add an identity provider following interactive prompts.
+
+[source,terminal]
+----
+$ rosa create idp --cluster=mycluster --interactive
+----
+
+[id="rosa-create-ingress_{context}"]
+== create ingress
+
+Adds an ingress endpoint to enable API access to the cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa create ingress --cluster=<clusterName> [arguments]
+----
+
+.Arguments
+|===
+|Option |Definition
+
+|-c, --cluster
+|(Required) The name or ID (string) of the cluster to which the ingress will be added.
+
+|--label-match
+|The label match (string) for ingress. The format must be a comma-delimited list of 'key=value'. If no label is specified, all routes are exposed on both routers.
+
+|--private
+|Restricts application route to direct, private connectivity.
+|===
+
+.Optional arguments inherited from parent commands
+|===
+|Option |Definition
+
+|-h, --help
+|Shows help for this command.
+
+|debug
+|Enables debug mode.
+
+|-i, --interactive
+|Enables interactive mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|-v, --v level
+|Log level for V logs.
+|===
+
+.Examples
+Add an internal ingress to a cluster named 'mycluster'.
+
+[source,terminal]
+----
+$ rosa create ingress --private --cluster=mycluster
+----
+
+Add a public ingress to a cluster named 'mycluster'.
+
+[source,terminal]
+----
+$ rosa create ingress --cluster=mycluster
+----
+
+Add an ingress with a route selector label match.
+
+[source,terminal]
+----
+$ rosa create ingress -c mycluster --label-match="foo=bar,bar=baz"
```

Comment:
Is there a reason that this one uses `-c` instead of `--cluster`? I'd be consistent across all the examples and use `--cluster`